web/remove-background-bulk/package.json

```
@@ -2,6 +2,7 @@
   "name": "batch-process",
   "version": "0.1.0",
   "private": true,
+  "homepage": "https://remove-background-bulk.vercel.app/apis/demos/bulk-remove-background",
```

```suggestion
  "homepage": "/apis/demos/bulk-remove-background",
```

Maybe we don't need it anymore, or we can use it without domain 🤔